src/test/java/org/gridsuite/ds/server/controller/DynamicSimulationControllerIEEE14Test.java

```diff
@@ -0,0 +1,169 @@
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.commons.datasource.ReadOnlyDataSource;
+import com.powsybl.commons.datasource.ResourceDataSource;
+import com.powsybl.commons.datasource.ResourceSet;
+import com.powsybl.dynamicsimulation.DynamicSimulationResult;
+import com.powsybl.dynamicsimulation.json.DynamicSimulationResultDeserializer;
+import com.powsybl.iidm.network.Importers;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.VariantManagerConstants;
+import com.powsybl.network.store.client.PreloadingStrategy;
+import com.powsybl.timeseries.TimeSeries;
+import org.gridsuite.ds.server.dto.dynamicmapping.Script;
+import org.gridsuite.ds.server.dto.timeseries.TimeSeriesGroupInfos;
+import org.gridsuite.ds.server.service.DynamicSimulationResultContext;
+import org.gridsuite.ds.server.service.client.dynamicmapping.DynamicMappingClientTest;
+import org.gridsuite.ds.server.service.client.timeseries.TimeSeriesClientTest;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.stream.binder.test.InputDestination;
+import org.springframework.cloud.stream.binder.test.OutputDestination;
+import org.springframework.messaging.Message;
+import org.springframework.test.web.reactive.server.EntityExchangeResult;
+import org.springframework.test.web.reactive.server.WebTestClient;
+import org.springframework.util.StreamUtils;
+import reactor.core.publisher.Mono;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Paths;
+import java.util.*;
+
+import static org.gridsuite.ds.server.service.parameters.ParametersService.MODELS_PAR;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+
+/**
+ * @author Thang PHAM <quyet-thang.pham at rte-france.com>
+ */
+public class DynamicSimulationControllerIEEE14Test extends AbstractDynamicSimulationControllerTest {
+    // mapping names
+    public static final String MAPPING_NAME_01 = "_01";
+
+    // directories
+    public static final String DATA_IEEE14_BASE_DIR = "/data/ieee14";
+    public static final String INPUT = "input";
+    public static final String OUTPUT = "output";
+    public static final String MODELS_GROOVY = "models.groovy";
+    public static final String RESULT_JSON = "result.json";
+
+    private static final String NETWORK_UUID_STRING = "11111111-0000-0000-0000-000000000000";
+    private static final String NETWORK_UUID_NOT_FOUND_STRING = "22222222-0000-0000-0000-000000000000";
+    private static final String VARIANT_1_ID = "variant_1";
+    private static final String NETWORK_FILE = "IEEE14.iidm";
+
+    private static final Map<UUID, List<TimeSeries>> TIME_SERIES_MOCK_BD = new HashMap<>();
```

Comment:
Why are you using a static attribute?

Reply:
You are right, this fake database is not necessary declared as static.. 
static is removed!